### PR TITLE
feat: disable/enable canary by clicking label

### PR DESCRIPTION
**Pull request by eldemcan**
instead of enable/disable canary env from console
we can now enable/disable by clicking Canary label

### CODE REVIEW

**Comment by eldemcan** [comment]
@svedova could you take a look at this please?

**Comment by eldemcan** [comment]
@svedova I manage to get to shot label just for admin but I can't seem to pass some tests. Could you please help me update those?

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -2,8 +2,10 @@ import React from "react";
 import cn from "classnames";
 import logoText from "~/assets/logos/stormkit-logo-text-h-white.svg";
 import logoIcon from "~/assets/logos/stormkit-logo-circle.svg";
+import { connect } from "~/utils/context";
+import AuthContext, { AuthContextProps } from "~/pages/auth/Auth.context";
 
-interface Props {
+interface Props extends AuthContextProps {
```
Comment (svedova) [inline src/components/Logo/Logo.tsx:8]:
```ts
interface Props extends Pick<AuthContextProps, "user"> {
```

File: src/pages/auth/Auth.context.tsx [hunk 2]
```diff
@@ -15,11 +15,11 @@ import {
...
-  error: string | null;
-  user: User;
-  accounts: Array<ConnectedAccount>;
-  loginOauth: (p: Provider) => Promise<LoginOauthReturnValue>;
-  logout: () => void;
+  error?: string | null;
+  user?: User;
+  accounts?: Array<ConnectedAccount>;
+  loginOauth?: (p: Provider) => Promise<LoginOauthReturnValue>;
+  logout?: () => void;
```
Comment (svedova) [inline src/pages/auth/Auth.context.tsx:22]: I guess there is no need to change the signature here, probably could be handled in a different way. I'll take a look.

**Comment by eldemcan** [comment]
closing in favor of https://github.com/stormkit-io/app-stormkit-io/pull/476

**Pull request closed without merging**